rmf_rviz_plugin/src/specified_time_action_plan.yaml

```diff
@@ -0,0 +1,8 @@
+action_plan:
```

Comment:
same as before, consider letting this config file live somewhere else

Reply:
i intended to put the yaml files in rmf_rviz_plugin as example files. for test cases i am developing, i place the yaml files in somewhere along the lines of demos/include/[world_name]/[world_name]_test_plan.yaml i think it would be cleaner to remove it, as long as corresponding documentation is added to point the user to the right place.

Perhaps ill leave it in for now. Once documentation is added, I will remove it.